translations/map_elements.tex/cs.po

```diff
@@ -113,8 +113,8 @@ msgid ""
 msgstr ""
 "\\subsection*{Popis Dílků Mapy\\index{Dílek Mapy}}\n"
 "Každý Dílek Mapy je rozdělen do 7 samostatných \\textbf{Polí}, která mohou vaši Hrdinové \\textbf{Navštívit}.\n"
-"Když se váš Hrdina přesune na Pole, musí ho Navštívit, pokud je to možné nebo\n"
-"nejprve začít \\pagelink{Combat}{Boj} proti nepřátelům, kteří ho střeží a Navštívit ihned po vyhraném boji.\n"
+"Když se váš Hrdina přesune na Pole, musí ho Navštívit, pokud je to možné, nebo\n"
+"nejprve začít \\pagelink{Combat}{Souboj} proti nepřátelům, kteří ho střeží, než ho Navštíví.\n"
```

Comment:
I think it's important to mention that it has to be Visited right after the combat. I wouldn't change that part